### PR TITLE
added ShowContainer component

### PR DESCRIPTION
**Pull request by elemee3**
imported ShowActivity and ActivityUsers to it, adjusted routes

### CODE REVIEW

File: src/pages/ShowContainer/ShowContainer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import ShowActivity from '../Activities/ShowActivity';
+import ActivityUsers from '../ActivityUsers/ShowActivityUsers';
+
+class ShowContainer extends Component {
+  render() {
+    {console.log(this.props)}
+    return (
+      <div>
+        SUP {this.props.user.email}
```
Comment (bwanicur) [inline src/pages/ShowContainer/ShowContainer.js]: is "SUP" cruft ?

File: src/services/AuthService.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import decode from 'jwt-decode'
 
 export default class AuthService {
 	constructor(domain) {
-		this.domain = process.env.REACT_APP_API_URL
+		this.domain = 'http://localhost:3001/'
```
Comment (bwanicur) [inline src/services/AuthService.js]: hard-coded localhost in here ? cruft ?

File: src/pages/ShowContainer/ShowContainer.js (hunk 1 above)
Comment (POWE88) [reply src/pages/ShowContainer/ShowContainer.js]: removed. Updating PR

File: src/services/AuthService.js (hunk 2 above)
Comment (POWE88) [reply src/services/AuthService.js]: Necessary for local work?

**Pull request merged**